app/src/main/java/com/master/contactsapp/MainActivity.kt

```diff
@@ -0,0 +1,262 @@
+                    phone = "123456789",
+                    address = "New York",
+                    imageRes = R.drawable.some_photo_from_internet
+                )
+            )
+        }
+
+    }
+
+    @Composable
+    fun RoundInitials(contact: Contact) {
+        val initials = contact.name.first().toString() + contact.familyName.first().toString()
+        Box(
+            modifier = Modifier.fillMaxWidth(),
+            contentAlignment = Alignment.Center
+        )
+        {
+            Icon(
+                modifier = Modifier.align(Alignment.Center),
+                painter = painterResource(id = R.drawable.circle),
+                contentDescription = null,
+                tint = Color.Gray
+            )
+            Text(
+                modifier = Modifier
+                    .align(Alignment.Center),
+                text = initials,
+                fontWeight = FontWeight.Bold,
+            )
+        }
+    }
+
+    @Composable
+    fun ShowPhoto(contact: Contact) {
+        val image = contact.imageRes?.let { painterResource(id = it) }
+        Box(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(vertical = 8.dp),
+            contentAlignment = Alignment.Center
+        )
+        {
+            image?.let {
+                Image(
+                    modifier = Modifier
+                        .size(96.dp, 48.dp)
+                        .align(Alignment.Center),
+                    painter = it,
+                    contentDescription = null,
+                    contentScale = ContentScale.FillBounds,
+                )
+            }
+        }
+    }
+
+    @Composable
+    private fun ShowFamilyName(contact: Contact) {
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(8.dp),
+            horizontalArrangement = Arrangement.Center
+        ) {
+            Text(
+                text = contact.familyName,
+                fontWeight = FontWeight.Bold,
+                textAlign = TextAlign.Center,
+                style = MaterialTheme.typography.h6,
+            )
+            if (contact.isFavorite) {
+                Image(
+                    painter = painterResource(id = android.R.drawable.star_big_on),
+                    contentDescription = null,
+                    modifier = Modifier
+                        .size(24.dp),
+                )
+            }
+        }
+
+    }
+
+    @Composable
+    private fun NameAndSurename(contact: Contact) {
+        Column(
+            modifier = Modifier.fillMaxWidth(),
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            Text(
+                text = "${contact.name} ${contact.surname ?: ""}",
+                fontWeight = FontWeight.Bold,
+                textAlign = TextAlign.Center,
+                style = MaterialTheme.typography.h6,
+                modifier = Modifier
+                    .fillMaxWidth(),
```

Comment:
```suggestion
                modifier = Modifier.fillMaxWidth(),
```
Если у Modifier 1 параметр, лучше писать на 1 строке, но если 2 и более - каждый новый с новой строки